fl4health/clients/basic_client.py

```diff
@@ -273,6 +278,10 @@ def fit(self, parameters: NDArrays, config: Config) -> Tuple[NDArrays, int, Dict
         # Update after train round (Used by Scaffold and DP-Scaffold Client to update control variates)
         self.update_after_train(local_steps, loss_dict)
 
+        # Check if we should run an evaluation with validation data after fit (used by FedDGGA)
```

Comment:
Very minor, but I'd just rephrase the parenthetical to (for example, this is used by FedDGGA). The only reason I suggest that, is I actually plan to use this functionality to expand our options for checkpointing as well 🙂

Reply:
Makes sense, and it's interesting you have other uses for this!